cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,16 @@
+describe("Login", () => {
+  it("can login with valid credentials", () => {
+    cy.visit("https://feycalion.github.io/social-media-client")
```

Comment:
```suggestion
    cy.visit("/")
```

You are testing local code, and Cypress will open it's own live server so you only need to refer to the name of the file as a local path instead of a deployed site.